kbase-extension/static/kbase/js/util/kbaseApiUtil.js

```diff
@@ -27,21 +25,41 @@ define([
     function getAppVersionTag () {
         var tag = Jupyter.narrative.sidePanel.$methodsWidget.currentTag;
         if (!tag) {
-            tag = "release";
+            tag = 'release';
         }
         return tag;
     }
 
-    /*
-     * Expects idList to be a list of app ids
-     * If tag is not present, it uses the currently configured tag
+    /**
+     * Returns a Promise that resolves into a list of app specs in the same order as the
+     * ids in idList. These get augmented with the "full_info" key that has the results of
+     * "get_method_full_info" from the Narrative Method Store service.
+     * @param {Array} idList - a list of app ids
+     * @param {String} tag - a tag, one of 'release', 'beta', 'dev'
      */
     function getAppSpecs (idList, tag) {
         if (!tag) {
             tag = getAppVersionTag();
         }
         var nms = new NarrativeMethodStore(Config.url('narrative_method_store'));
-        return Promise.resolve(nms.get_method_spec({ids: idList, tag: tag}));
+        return Promise.all([
```

Comment:
Updated this to also include the `get_method_full_info` call. This is used to get info that the app cell uses, and it seemed appropriate to put it here as well.

Reply:
Some of the info is redundant, though, even in the existing app cell. That should get worked out in another PR.